test/integration/apiserver/apply_test.go

```diff
@@ -63,3 +64,38 @@ func TestApplyAlsoCreates(t *testing.T) {
 		t.Fatalf("Failed to retrieve object: %v", err)
 	}
 }
+
+// TestCreateOnApplyFailsWithUID makes sure that PATCH requests with the apply content type
+// will not create the object if it doesn't already exist and it specifies a UID
+func TestCreateOnApplyFailsWithUID(t *testing.T) {
+	// TODO: make setup do this optionally, and make it temporary rather than affecting global state.
```

Comment:
SetFeatureGateDuringTest will set a gate and give you a function you can call deferred to reset the gate to its original value